drawBot/drawBotDrawingTools.py

```diff
@@ -995,7 +1056,13 @@ def cmykShadow(self, offset, blur=None, color=None):
         self._requiresNewFirstPage = True
         self._addInstruction("cmykShadow", offset, blur, color)
 
-    def linearGradient(self, startPoint=None, endPoint=None, colors=None, locations=None):
+    def linearGradient(
+        self,
+        startPoint: Point | None = None,
+        endPoint: Point | None = None,
+        colors: list[RGBColor] | None = None,
```

Comment:
`RGBColor` is defined as `float | tuple[float, ...]`, but a list of floats does not work here.

Reply:
Mmm, I understand what you mean. Colors should be tuples of 3 or 4 floats, correct?
Then we need to change the docstring too as 
> * `colors` as a list of colors, described similary as `fill`
it's not entirely correct

Reply:
fixed with 1bdd28d3da63a9a7b50d4de304cfab5efbe08e5d